engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -92,6 +100,51 @@ public void bind() {
     }
 
     /**
+     * Once an FBO is bound, opengl commands will act on it, i.e. by drawing on it.
+     * Meanwhile shaders might output not just colors but additional per-pixel data. This method establishes on which
+     * of an FBOs attachments, subsequent opengl commands and shaders will draw on.
+     *
+     * @param c If True the color buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param n If True the normal buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     * @param l If True the light buffer is set as drawable. If false subsequent commands and shaders won't be able to draw on it.
+     */
+    public void setRenderBufferMask(boolean c, boolean n, boolean l) {
+        if (this.color == c && this.normal == n && this.lightBuffer == l) {
+            return;
+        }
+
+        this.color = c;
+        this.normal = n;
+        this.lightBuffer = l;
+
+        int attachmentId = 0;
+
+        IntBuffer bufferIds = BufferUtils.createIntBuffer(3);
+
```

Comment:
Let's add:

// TODO: change GL_COLOR_ATTACHMENT0_EXT + attachmentId into something like COLOR_BUFFER_ATTACHMENT, in turn set within the class or method